src/sync/index.js

```diff
@@ -205,7 +206,10 @@ const startSync = async () => {
     await DBHelper.updateEventStatusOracleResultSetting(blockTime);
     await DBHelper.updateEventStatusOpenResultSetting(blockTime);
     await DBHelper.updateEventStatusArbitration(blockTime);
-    await DBHelper.updateEventStatusWithdrawing(blockTime);
+    // await DBHelper.updateEventStatusWithdrawing(blockTime);
```

Comment:
undo this and keep it separate. set a var newWithdrawEvents then pass it to `updateLeaderboards({ newWithdrawEvents, syncPromises, limit })`